pylint/checkers/nested_min_max.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Check for use of nested min/max functions."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages, safe_infer
+from pylint.interfaces import INFERENCE
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+
+class NestedMinMaxChecker(BaseChecker):
+    FUNC_NAMES = ("builtins.min", "builtins.max")
+
+    name = "nested_min_max"
+    msgs = {
+        "W3201": (
+            "Do not use nested call of '%s'  it's possible to do '%s' instead",
```

Comment:
```suggestion
            "Do not use nested call of '%s' it's possible to do '%s' instead",
```

Reply:
> 

```suggestion
            "Do not use nested call of '%s';  it's possible to do '%s' instead",
```

Reply:
In Dutch `it's possible` should be capitalised as the `;` is considered a period. I'm not sure about English in this case 😅

Reply:
Oh 😅. I'm not living in NL long enough to know that but I think it would be lowercase in English. Its OK with me either way!